cli/tests/testdata/run/config_file_lock_custom.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lock": "run/lock_check_err2.json"
```

Comment:
This indicates a bug. The lockfile path should be relative to the configuration file and not the current working directory as this seems to be.

Reply:
Fixed!